packages/space-opera/src/components/shared/expandable_content/expandable_section.css.ts

```diff
@@ -25,16 +25,13 @@ export const styles: CSSResult = css`:host {
   max-height: 0;
   overflow: hidden;
   padding: 0 20px;
-  transition-duration: 300ms;
-  transition-timing-function: ease-out;
 }
 
 .SectionContent[open] {
   background-color: var(--me-theme-container-background);
-  overflow: hidden;
+  overflow: unset;
```

Comment:
These changes don't seem related. Are they in the original PR?

Reply:
No, that's from reverting chris' recent PR, which broke some tests because they weren't being run.